src/odemis/gui/win/acquisition.py

```diff
@@ -27,29 +27,34 @@
 import os.path
 from builtins import str
 from concurrent.futures._base import CancelledError
+from typing import List, Tuple
 
 import numpy
-import wx
 
 import odemis.gui.model as guimodel
-from odemis import model, dataio
-from odemis.acq import stream, path, acqmng, stitching
-from odemis.acq.stitching import FocusingMethod, WEAVER_MEAN, REGISTER_IDENTITY, acquireOverview
-from odemis.acq.stream import EMStream, NON_SPATIAL_STREAMS, OpticalStream, ScannedFluoStream, LiveStream
-from odemis.gui.acqmng import presets, preset_as_is, apply_preset, \
-    get_global_settings_entries, get_local_settings_entries
+import wx
```

Comment:
why is `import wx` here and not grouped with numpy? If there is a specific reason, can you add a comment?

Reply:
No, it just got automatically reordered by my IDE when adding the new import. I'll put it back where it belongs.